yt_dlp/extractor/playsuisse.py

```diff
@@ -0,0 +1,121 @@
+# coding: utf-8
+import json
+
+from .common import InfoExtractor
+from ..utils import int_or_none
+
+
+class PlaySuisseIE(InfoExtractor):
```

Comment:
```suggestion
class PlaySuisseIE(InfoExtractor):
    IE_DESC = 'playsuisse.ch'
```
This PR doesn't work for me without this change. Otherwise I get this error:
```
ERROR: 'PlaySuisseIE' object has no attribute 'IE_DESC'
```
I'm not sure if that's the right approach though (@pukkandan ?)

Reply:
You must be applying the PR on a commit before https://github.com/yt-dlp/yt-dlp/commit/231025c4632d7a48540b18997c715a8df8a6ff8e